package.json

```diff
@@ -88,8 +89,10 @@
     "websocket": "^1.0.35"
   },
   "scripts": {
-    "format": "prettier --write src/**/*.{js,jsson,md} docs/*.md webpack/*.js bin/*.js",
```

Comment:
seems like /**/*.{} syntax is not working, I'll remove it and format the whole folders